modules/ROOT/pages/api-groups-creating-groups.adoc

```diff
@@ -49,15 +51,17 @@ The *Creating a new group* page is displayed. You are now ready to <<adding-api-
 [[adding-api-instances-to-groups]]
 == Add API instances to API Groups
 
-Your API Group is not complete and functional unless you add the API instances to the group that you want to bundle. You can add API instances that belong in the same business group or organizations.
+Your API Group is not complete and functional unless you add the API instances to the group that you want to bundle. When you add API instances to an API Group, you can add:
 
-Note that you can add only those API instances that belong the same environment as the API Group instance.
+* Only those API instances that belong in the same business group or organizations.
+* Only those API instances that belong the same environment as the API Group instance.
+* Only those API instances that belong to the same identity provider (IdP) as the API Group instance. You cannot mix IdPs in an API Group instance.
```

Comment:
This is correct but the group instance does not have an Idp, it is set by its instances. So I would reformulate this
```suggestion
* Only those API instances that belong to the same identity provider (IdP). You cannot mix IdPs in an API Group instance.
```

Reply:
Done! Thanks!